src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1279,14 +1278,17 @@ private Deadline NewAckDeadline()
             return Deadline.Now + Settings.SysMsgAckTimeout;
         }
 
-        private void PublishAndThrow(Exception reason, LogLevel level)
+        private void PublishAndThrow(Exception reason, LogLevel level, bool needToThrow = true)
```

Comment:
This method is used all over including the OneForOneStrategy of the EndpoingWriter. So that not to break the old logic, I added a default parameter _bool needToThrow = true_ and turned off throwing inside the strategy. The exception is just escalated....